examples/vision/integrated_gradients.py

```diff
@@ -0,0 +1,474 @@
+"""
+Title: Integrated Gradients
+Author: [A_K_Nain](https://twitter.com/A_K_Nain)
+Date created: 2020/06/2
+Last modified: 2020/06/2
+Description: How to obtain integrated gradients for a classification model.
+"""
+
+"""
+## Integrated Gradients
+[Integrated Gradients](https://arxiv.org/abs/1701.07875) is a technique for
+attributing a classification model's prediction to its input features. It is
+one of the interpretibility techniques used to understand why did the model make
+this prediction on this input.
+
+[Integrated Gradients](https://arxiv.org/abs/1701.07875) involves following steps:
+1. Construct a sequence of images interpolating from a baseline to the actual image.
+2. Do a forward pass on these interpolated images and get the gradients.
+2. Average the gradients across these images.
+
+Refrences:
+Paper: https://arxiv.org/abs/1701.07875
+Original implementation: https://github.com/ankurtaly/Integrated-Gradients
+"""
+
+"""
+## Setup
+"""
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import ndimage
+from pathlib import Path
+from IPython.display import Image
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+from tensorflow.keras.applications.xception import (
```

Comment:
In this case, you can simply import `applications.xception` then access the different objects from the module.